src/components/index.jsx

```diff
@@ -588,7 +588,7 @@ class MaterialUiPhoneNumber extends React.Component {
     const {
       classes, dropdownClass, localization, disableDropdown, native,
     } = this.props;
-    const inputFlagClasses = `flag ${selectedCountry.iso2}`;
+    const inputFlagClasses = `flag ${selectedCountry && selectedCountry.iso2}`;
```

Comment:
This will output the string "false" if selectedCountry is undefined. So it's not the preferred solution. I will fix it for you.